RecoTracker/MkFit/plugins/HitSelectionWindows2017.h

```diff
@@ -0,0 +1,5868 @@
+#ifndef HitSelectionWindows_h
+#define HitSelectionWindows_h
+
+namespace mkfit {
+
+  struct HitSelectionWindows2017 {
+  public:
+    float m_dp_params[9][72][3];
+    float m_dq_params[9][72][3];
+    float m_c2_params[9][72][3];
```

Comment:
I suspect it would be better to store these hardcoded constants as (constant) global variables than construct `HitSelectionWindows` once for each iteration.